examples/log-attack2/index.rsh

```diff
@@ -0,0 +1,28 @@
+'reach 0.1';
+const LogAttack2 = {
+    m2: Fun([Address, UInt], Null),
+    m3: Fun([UInt], Null),
+}; 
+export const main = Reach.App(() => {
+  const Alice = Participant('Alice', {
+    ...hasConsoleLogger,
+    m2: Fun([], Tuple(Address, UInt)),
+    m3: Fun([], UInt),
+  });
+  deploy();
+    Alice.only(() => {
+      const [acct, amt] = declassify(interact.m2()); 
+      const date = declassify(interact.m3()); 
+    });
+    Alice.publish(acct, amt, date);
+    commit();
+    Alice.publish();
+    const log2 = remote(acct, LogAttack2);
```

Comment:
This should be like the previous file but the remote Solidity should emit the NEXT Reach event